src/scripts/viewer/2d/graphViewer.js

```diff
@@ -12,7 +12,8 @@ function graphViewer() {
       'source': '=',
       'nodeSelected': '=',
       'root': '=',
-      'mode': '='
+      'mode': '=',
+      'showSize': '='
```

Comment:
I thought that `showSize` was going to show up from the parent `$scope` but this didn't work. Any ideas?